monai/networks/blocks/dints_block.py

```diff
@@ -0,0 +1,107 @@
+# Copyright 2020 - 2021 MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import torch
+import torch.nn as nn
+
+__all__ = ["FactorizedIncreaseBlock", "FactorizedReduceBlock", "P3DReLUConvBNBlock", "ReLUConvBNBlock"]
+
+
+class FactorizedIncreaseBlock(nn.Module):
```

Comment:
Maybe we can change the class name to be more general?  Does `scale_factor` must be `2`? If not, can we make the `scale_factor`,  inerpolation `mode` and `align_corners` as configurable parameters?

Thanks.

Reply:
Currently the `scale_factor` is fixed as 2 in the algorithm. other values have not been used before.